src/test.js

```diff
@@ -785,12 +785,12 @@ function internalStart( test ) {
 }
 
 function numberOfTests( module ) {
-	let count = module.tests.length,
-		modules = [ ...module.childModules ];
+	let count = module.tests.length;
+	const modules = [ ...module.childModules ];
 
 	// Do a breadth-first traversal of the child modules
 	while ( modules.length ) {
-		let nextModule =  modules.shift();
+		const nextModule =  modules.shift();
 		count += nextModule.tests.length;
 		modules.push( ...nextModule.childModules );
 	}
```

Comment:
of course! It's indeed not related (I'm sorry for the noise), and I'm always in favor of follow up PRs rather than blocking anything for too long.